extras/imaging/examples/hdui/dataSourceValueTreeView.cpp

```diff
@@ -56,7 +57,7 @@ class Hdui_ValueItemModel : public QAbstractItemModel
             if (index.column() == 0) {
                 std::ostringstream buffer;
                 buffer << _value;
-                return QVariant(buffer.str().data());
+                return QVariant(QLatin1String(buffer.str().data()));
```

Comment:
Should we use a string type that supports UTF-8 identifiers? Tagging @erslavin for thoughts

Reply:
Good point; in fact it turns out that using UTF-8 strings is actually what Qt5 is doing under the hood in those cases (see the [Qt5 documentation for QVariant](https://doc.qt.io/qt-5/qvariant.html#QVariant-11), confirmed by a Qt employee). We've changed it to do the same, just explicitly, so that it works for both Qt5 and Qt6, and regardless of whether `QT_NO_CAST_FROM_ASCII` is set or not.